### ISSUE TITLE
Support usage "as much as possible" for this agent without having to specify it

### ISSUE THREAD

**Issue by shushry** [jenkinsci/amazon-ecs-plugin#236]
The real power in this plugin would be the ability to spin up agents on demand to jobs that need a place to run - without having to specify the agent label specifically to direct a Jenkins project at it.    We have a large library of existing jobs, that are set to "agent=any" or no specific restriction.  My expectation is that Jenkins would be able to spin up an agent as configured for a job of this type - without having to specifically restrict it to this cloud configuration.   Currently projects must specifically be restricted to use the agent label given during the setup of this plugin, in order to make use of it.

In short, support the use of "agent = any" or "use this agent as much as possible" functionality in Jenkins to spin up agents here in ECS as needed.
